demos/README.md

```diff
@@ -31,7 +31,7 @@ bazel run //demos:upload_prod_demo
 
 1. Clone `https://github.com/pixie-io/microservice-kafka` and switch to the `pixie` branch.
 
-2. (optional) Build the container images & update the individual yaml files.
+2. (optional) Build the container image & update the individual yaml files.
```

Comment:
Is this intentional? There were three microservices in that demo if I recall correctly.

Reply:
Yea, we are sharing images now since the client and server are all bundled within the same image and the image size was ~500mb.

https://github.com/pixie-io/pixie/pull/607#discussion_r988355871